extern/storage-sealing/input.go

```diff
@@ -187,10 +188,13 @@ func (m *Sealing) handleAddPiece(ctx statemachine.Context, sector SectorInfo) er
 		offset += padLength.Unpadded()
 
 		for _, p := range pads {
+			expectCid := zerocomm.ZeroPieceCommitment(p.Unpadded())
```

Comment:
Shouldn't this be `deal.deal.DealProposal.PieceCID` ? Why are we self calculating the expected PieceCid here ?